Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageDeliveryStatusCheckmarkView.swift

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// A view that displays message delivery receipts.
+open class ChatMessageDeliveryStatusCheckmarkView: _View, ThemeProvider {
+    /// The data this view component shows.
+    open var content: MessageDeliveryStatus? {
+        didSet { updateContentIfNeeded() }
+    }
```

Comment:
It is better to create a `Content` struct for the views, even if it only has 1 property. This will make sure we can add new data in the future

Reply:
It makes sense if init is kept internal. Let me add it 👍

Reply:
Done ✅